Quantum-Control-Applications/ESR/signal_test.py

```diff
@@ -8,31 +8,36 @@
 from qm.QuantumMachinesManager import QuantumMachinesManager
 from configuration import *
 import matplotlib.pyplot as plt
-from qm.simulate.credentials import create_credentials
+from macros import get_c2c_time
 
 ###################
 # The QUA program #
 ###################
 
-pulse1_len = 400 // 4
-pulse2_len = pulse1_len * 2
-pulse_delay = 2000 // 4 - pulse1_len / 2 - pulse2_len / 2
-readout_delay = 2000 // 4 - pulse2_len / 2 - readout_len // 8 - 8
+pulse1_len = 400 // 4  # Pi/2 pulse
+pulse2_len = pulse1_len * 2  # Pi pulse
 
-cooldown_time = int(10e6 // 4)
+cooldown_time = 10 * u.ms // 4  # Resonator or qubit relaxation time
+safe_delay = 2 * u.us // 4  # Delay to safely avoid sending pulses during measurement windows
+
+# Center to center time between first and second pulse
+pulse_delay = safe_delay - (pulse1_len + pulse2_len) // 2 - 0
```

Comment:
yeah, the zeros look ugly. Maybe if we want to keep them we can put variables. At the end of the installationwe had variables that were updated after simulated and put into the correct place to reload the qm and then the experiment c orreclty. Another scenario is not to put any delay and catch during the program indeed. If it is zero for this file that is a pre-defined progarm and will likely not get moidfy to make a more  complex experiemtn then I don't think we need 0

Reply:
Ok I agree !